src/file_utils.c

```diff
@@ -33,3 +34,38 @@ int create_file_name(char *new_name, const size_t name_size, const char *templat
 
 	return 0;
 }
+
+/**
```

Comment:
double comments - in C and in H
also, maybe add 
@param **[in]**  f
@param **[out]** size output for file size

and sort - input params at start, ouput at the end. I'm not sure if we do have it in coding standard. but keeping output params at end of the params list makes the code more readable